### DISCUSSION TITLE
Allow grouping of saved connections

### DISCUSSION THREAD

**Discussion by hsols** [beekeeper-studio/beekeeper-studio#1932] [Ideas]
Would be grand having the ability to organize saved connections within a group/folder structure, e.g. when organising connections per project 🫶

**Comment by jlanzarotta** [comment]
I would like this feature as well. Is there a way to do this?

**Reply by twysto** [reply]
You can check the status of this issue and let a comment if you feel it necessary:
https://github.com/beekeeper-studio/beekeeper-studio/issues/799

Also you can let a comment here on this subreddit:
https://www.reddit.com/r/SQL/comments/1ctsmtr/what_feature_should_i_add_next_to_beekeeper_studio/
